### PR TITLE
Qt-downloader latest with gha testing

### PR DESCRIPTION
**Pull request by RAJAGOPALAN-GANGADHARAN**

### CODE REVIEW

**Comment by annulen** [comment]
Seems like GitHub disabled set-env and add-path commands, see https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/. Workflow file needs to be changed

**Comment by RAJAGOPALAN-GANGADHARAN** [comment]
Sure. Will look into it

File: .github/workflows/build_cmake.yml [hunk 1]
```diff
@@ -153,13 +153,12 @@ jobs:
...
-        echo "::set-env name=CCACHE_BASEDIR::$GITHUB_WORKSPACE"
-        echo "::set-env name=CCACHE_DIR::$GITHUB_WORKSPACE/.ccache"
-        echo "::set-env name=cmake_args::\
-            -DCMAKE_C_COMPILER_LAUNCHER=ccache \
-            -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
-            -DENABLE_PCH=OFF"
+        echo "$GITHUB_WORKSPACE" >> $GITHUB_PATH
+        echo "CCACHE_BASEDIR=$GITHUB_WORKSPACE" >> $GITHUB_ENV
+        echo "CCACHE_DIR=$GITHUB_WORKSPACE/.ccache" >> $GITHUB_ENV
+        echo "cmake_args=-DCMAKE_C_COMPILER_LAUNCHER=ccache " \
```
Comment (annulen) [inline .github/workflows/build_cmake.yml:159]: Does resulting `cmake_args` contain `-DCMAKE_CXX_COMPILER_LAUNCHER=ccache` and `-DENABLE_PCH=OFF`? Unescaped spaces make me feel uneasy.

Comment (RAJAGOPALAN-GANGADHARAN) [reply .github/workflows/build_cmake.yml:159]: https://github.com/qtwebkit/qtwebkit/pull/1038/checks?check_run_id=1545151212#step:13:19

The env variable is rightly set, check put ubuntu for e.g.

**Comment by annulen** [comment]
Merged

**Pull request closed without merging**